launch/boson.launch

```diff
@@ -32,8 +32,4 @@
     <param name="camera_info_url" type="str" value="$(arg camera_info_url)"/>
   </node>
 
-  <!-- Optional: built-in image view opens window to view the streaming feed -->
-  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
```

Comment:
sure we should be removing this?

Reply:
we won't want to run image_view by default typically, right? we could run rviz to view images.